src/plugins/intel_npu/tools/protopipe/src/parser/config.cpp

```diff
@@ -514,10 +541,33 @@ struct convert<OpDesc> {
             type = "Compound";
         }
         if (type == "Infer") {
+            const std::set<std::string> parameters = {"config", "device", "framework", "il", "iml",
```

Comment:
The point was to specify only that you can't use `CPU` parameters when you using `Infer` type. There is no need to check twice for other parameters as all possible parameters can be checked first in the `OpDesc`. Please refer to the global comment and redo. If you don't understand that comment, lets sync.